frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -52,18 +55,22 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const countryList = await getLookupService().getAllCountries();
   const regionList = await getLookupService().getAllRegions();
   const preferredLanguage = personalInformation.preferredLanguageId ? await getLookupService().getPreferredLanguage(personalInformation.preferredLanguageId) : undefined;
-
+  const maritalStatusList = await getLookupService().getAllMaritalStatuses();
+  const dateString = personalInformation.birthDate ? new Date(personalInformation.birthDate).toLocaleDateString() : undefined;
+  const birthParsedFormat = dateString && isValidDate(dateString, 'mm/dd/yyyy', '/') ? toLocaleDateString(await parse(dateString, 'dd/mm/yyyy', new Date()), await locale) : undefined;
```

Comment:
I think you can remove the `await` from `locale` here since you're awaiting the `getLocale()` call.

Reply:
Fixed: 2ab2c0455279953194aad59443cee505be0bec8d